shared/wallets/wallet/settings/popups/really-remove-account/container.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import {connect, type TypedState} from '../../../../../util/container'
+import * as Constants from '../../../../../constants/wallets'
+import * as ConfigGen from '../../../../../actions/config-gen'
+import * as WalletsGen from '../../../../../actions/wallets-gen'
+import * as Types from '../../../../../constants/types/wallets'
+import ReallyRemoveAccountPopup from '.'
+
+const mapStateToProps = (state: TypedState, {routeProps}) => {
+  const accountID = routeProps.get('accountID')
+  const secretKey = Constants.getSecretKey(state, accountID).stringValue()
+
+  return {
+    accountID,
+    secretKey,
+    name: Constants.getAccount(state, accountID).name,
+  }
+}
+const mapDispatchToProps = (dispatch: Dispatch, {navigateUp}) => ({
+  _onClose: () => dispatch(navigateUp()),
```

Comment:
Since `ReallyRemoveAccount` is a popup on top of the `RemoveAccount` popup, then hitting cancel on `ReallyRemoveAccount` will take a user back to the `RemoveAccount` popup which I don't think is what we want. See my earlier comment for more explanation.